packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/messages.g.h

```diff
@@ -42,6 +157,109 @@ NS_ASSUME_NONNULL_BEGIN
+                       removing:(NSArray<NSString *> *)idsToRemove
+                          error:(FlutterError *_Nullable *_Nonnull)error;
+/// Updates the set of tile overlays on the map.
+- (void)updateTileOverlaysByAdding:(NSArray<FGMPlatformTileOverlay *> *)toAdd
+                          changing:(NSArray<FGMPlatformTileOverlay *> *)toChange
+                          removing:(NSArray<NSString *> *)idsToRemove
+                             error:(FlutterError *_Nullable *_Nonnull)error;
+/// Gets the screen coordinate for the given map location.
+///
+/// @return `nil` only when `error != nil`.
+- (nullable FGMPlatformPoint *)screenCoordinatesForLatLng:(FGMPlatformLatLng *)latLng
+                                                    error:(FlutterError *_Nullable *_Nonnull)error;
+/// Gets the map location for the given screen coordinate.
+///
+/// @return `nil` only when `error != nil`.
+- (nullable FGMPlatformLatLng *)latLngForScreenCoordinate:(FGMPlatformPoint *)screenCoordinate
+                                                    error:(FlutterError *_Nullable *_Nonnull)error;
+/// Gets the map region currently displayed on the map.
+///
+/// @return `nil` only when `error != nil`.
+- (nullable FGMPlatformLatLngBounds *)visibleMapRegion:(FlutterError *_Nullable *_Nonnull)error;
+/// Moves the camera according to [cameraUpdate] immediately, with no
+/// animation.
+- (void)moveCameraWithUpdate:(FGMPlatformCameraUpdate *)cameraUpdate
+                       error:(FlutterError *_Nullable *_Nonnull)error;
+/// Moves the camera according to [cameraUpdate], animating the update.
+- (void)animateCameraWithUpdate:(FGMPlatformCameraUpdate *)cameraUpdate
+                          error:(FlutterError *_Nullable *_Nonnull)error;
+/// Gets the current map zoom level.
+///
+/// @return `nil` only when `error != nil`.
+- (nullable NSNumber *)currentZoomLevel:(FlutterError *_Nullable *_Nonnull)error;
+/// Show the info window for the marker with the given ID.
+- (void)showInfoWindowForMarkerWithIdentifier:(NSString *)markerId
+                                        error:(FlutterError *_Nullable *_Nonnull)error;
+/// Hide the info window for the marker with the given ID.
+- (void)hideInfoWindowForMarkerWithIdentifier:(NSString *)markerId
+                                        error:(FlutterError *_Nullable *_Nonnull)error;
+/// Returns true if the marker with the given ID is currently displaying its
+/// info window.
+///
+/// @return `nil` only when `error != nil`.
+- (nullable NSNumber *)
+    isShowingInfoWindowForMarkerWithIdentifier:(NSString *)markerId
+                                         error:(FlutterError *_Nullable *_Nonnull)error;
+/// Sets the style to the given map style string, where an empty string
+/// indicates that the style should be cleared.
+///
+/// If there was an error setting the style, such as an invalid style string,
+/// returns the error message.
+- (nullable NSString *)setStyleStyle:(NSString *)style
```

Comment:
Not sure how to avoid this in the generated code, but `setStyleStyle:style` isn't the most clear...

Reply:
Oops, missed that since I let Xcode auto-create all the protocol conformance stubs for me. It's easy to fix in `messages.dart`.

(I'll also file a Pigeon issue, because I thought we'd special-cased the logic to avoid duplication in the default name when it's the same as the end of the method name.)